include/TPP/Dialect/Xsmm/XsmmOps.td

```diff
@@ -117,11 +117,11 @@ def Xsmm_TernaryDispatchOp : Xsmm_Op<"ternary.dispatch",[Pure]> {
   }];
 
   let arguments = (ins Xsmm_TernaryKind:$kind, DenseI64ArrayAttr:$inputs,
-                       Xsmm_DataType:$dataType);
+                       Xsmm_DataType:$dataType, BoolAttr:$isVNNI);
   let results = (outs I64:$results);
 
   let assemblyFormat = [{
-    $kind $inputs `(` `dataType` $dataType `)` attr-dict
+    $kind $inputs `(` `dataType` $dataType `isVNNI` $isVNNI `)` attr-dict
```

Comment:
I would add a comma in between, like: `(` `dataType` $dataType `,` `isVNNI` $isVNNI `)`. Do we also expect to have this flag for other dispatches?

Reply:
No, this flag is only for ternary dispatches.

Reply:
Ok perfect, I also assume this flag is only valid for BF16? Do we check this?

Reply:
we don't, I'll add an assert in the dispatch method for this.